packages/common/src/types/index.ts

```diff
@@ -162,13 +183,13 @@ export type Sandbox = {
   originalGit: GitInfo | undefined;
   originalGitCommitSha: string | undefined;
   originalGitChanges:
-    | {
-        added: Array<string>;
-        modified: Array<string>;
-        deleted: Array<string>;
-        rights: 'none' | 'read' | 'write' | 'admin';
-      }
-    | undefined;
+  | {
+    added: Array<string>;
+    modified: Array<string>;
```

Comment:
@SaraVieira This can just be

```suggestion
    modified: string[];
```

Reply:
Changed